csharp-advanced-topics/DelegatesInCsharp/ActionFuncTests/ShapeTests.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+
+namespace ActionFunc.Tests
+{
+    [TestClass()]
+    public class ShapeTests
+    {
+        [TestMethod()]
+        public void PrintTest()
```

Comment:
Same review as the other test class. vars, (), try-catch, use newlines between Arrange/Act/Assert.